websites/W/Warriorcats/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Warriorcats",
+	"description": {
+		"en": "meow"
+	},
+	"url": "warriorcats.de",
```

The `www` subdomain is supported by the website, so it should be added here as well.
```suggestion
	"url": [
		"warriorcats.de",
		"www.warriorcats.de"
	],
```